unit-tests/native/src/test/scala/org/scalanative/testsuite/posixlib/sys/UtsnameTest.scala

```diff
@@ -21,39 +21,39 @@ class UtsnameTest {
       0
     )
 
-    val sysname = u.at1.asInstanceOf[Ptr[CChar]]
-    val nodename = u.at2.asInstanceOf[Ptr[CChar]]
-    val release = u.at3.asInstanceOf[Ptr[CChar]]
-    val version = u.at4.asInstanceOf[Ptr[CChar]]
-    val machine = u.at5.asInstanceOf[Ptr[CChar]]
+    val sysname = u._1.asInstanceOf[CArray[Byte, _256]]
```

Comment:
Thank you for this iteration.

posixlib has one foot in C land and the other in Java Land,
so things get complex.

I suggest leaving these as `Ptr[CChar]]`, in general and especially in Test code.  That is, treat them as CStrings
and not as CArray (of known size).

The Open Group 2018 standard says:
```
The character arrays are of unspecified size, but the data stored in them shall be terminated by a null byte.
```

Now in Test code, there may be some cause to set a maximum size tested, say 1024 or such, to ensure
that the OS did not return an "unreasonable" or non-null-terminated string.  I will have to look at the rest of the code
in this file.

Am I missing a place where the magic _256 is defined.  The
OS could have returned larger than that. I think it is 
OS dependent.

Reply:
I think I would also use the Ops name, rather than _1, just
to make sure they get exercised.  If something _can_ go
wrong, it will.

Then again, I am not the one writing this PR and must 
defer to the author.

Reply:
Thank you for the feedback and for addressing it in another issue. I will check it out when I have time.

Reply:
mox692.   Any improvements are welcome, just want to re-assure you that just because you touched it....

If you have time and motivation, I think the "short fix" (change to _257 and give Open Group, man 7, and
issue) is cost effective.  Leaving the Issue open pending the proper long term fix may be useful, so that
the "proper" fix is not forgotten.

I hope you had a good holiday weekend.